h/models/job.py

```diff
@@ -47,6 +49,11 @@
 class Job(Base):
     """A job in the job queue."""
 
+    class JobName(str, Enum):
+        SYNC_ANNOTATION = "sync_annotation"
+        ANNOTATION_SLIM = "annotation_slim"
+        PURGE_USER = "purge_user"
```

Comment:
Moved this enum from the service onto the model and added `PURGE_USER`.

Enums on services get mocked in the unit tests of other services which is a pain (e.g. SQLAlchemy crashes when trying to write mocks to the DB). I think the model is probably a better place for this enum anyway: `JobQueueService` may not be the only code that needs to work directly with the job table.

Note that the existing `SYNC_ANNOTATION` and `ANNOTATION_SLIM` in this enum aren't used anyway, everywhere just uses the strings directly!